provider/marathon/marathon.go

```diff
@@ -369,6 +449,62 @@ func (p *Provider) hasCircuitBreakerLabels(application marathon.Application) boo
 	return ok
 }
 
+// Map of services properties
+// we can get it with label[serviceName][propertyName] and we got the propertyValue
+type servicePropertyValues map[string]string
+
+type labelServiceProperties map[string]servicePropertyValues
+
+// Check if for the given application, we find labels that are defining services
+func (p *Provider) hasServices(application marathon.Application) bool {
```

Comment:
The current template doesn't use it though. Unless we see a concrete use case for the future, I'd rather drop the method for now.